draft-ietf-tls-dtls-rrc.md

```diff
@@ -152,6 +153,55 @@ cookie is a 8-byte field containing arbitrary data.
 The `return_routability_check` message MUST be authenticated and encrypted using
 the currently active security context.
 
+# Off-Path Packet Forwarding {#off-path}
+
+An off-path attacker that can observe packets might forward copies of
+genuine packets to endpoints. If the copied packet arrives before
+the genuine packet, this will appear as a NAT rebinding. Any genuine
+packet will be discarded as a duplicate. If the attacker is able to
+continue forwarding packets, it might be able to cause migration to a
+path via the attacker. This places the attacker on-path, giving it
+the ability to observe or drop all subsequent packets.
+
+This style of attack relies on the attacker using a path that has
+approximately the same characteristics as the direct path between
+endpoints. The attack is more reliable if relatively few packets are
+sent or if packet loss coincides with the attempted attack.
+
+A non-path probing packet received on the original path that increases the
+maximum received packet number will cause the endpoint to move back
+to that path. Eliciting packets on this path increases the
+likelihood that the attack is unsuccessful. Therefore, mitigation of
+this attack relies on triggering the exchange of packets.
+
+In response to an apparent migration, endpoints MUST validate the
```

Comment:
Is this a MUST or a SHOULD?  If it's a MUST it's probably better to describe along  with the path validation algorithm?